website/docs/basics/blocks-component-in-block.md

```diff
@@ -287,3 +287,58 @@ This attribute is available in the options component only. You can't set it via
 ### How do my example attributes behave when I use components in blocks?
 
 Exactly the same way as attributes.
+
+### I have a component name that has multiple strings in a name, will this work?
+
+By multiple strings in a name we mean for example `jumbotron-cta` component name. The problem here is that we define components with dash and attributes by camelcase so you need to fix this. We have provided you with the helper to camelcase all your componentName strings.
+
+**Example:**
+
+You have a component named `jumbotron-cta` and you want to use it in your block. As described on this page, you will create a `components` key in your `manifest.json` and use `jumbotron-cta`.
+
+```json
+{
+	"components": {
+		"jumbotron-cta": "jumbotron-cta"
+	}
+}
+```
+
+But in all places in your component where you use `setAttributes` you must use [camelize helper](https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/helpers/camelize.js)
+
+```js
+
+import { camelize } from '@eightshift/frontend-libs/scripts/helpers';
+
+<ToggleControl
+	 label={'Custom Label'}
+	 onChange={(value) => setAttributes({ [`${camelize(componentName)}Use`]: value })}
+/>
+```
+
+### I have a component that I want to use manually
+
+We created this cool way of importing attributes and loading components in your blocks or other components, but that doesn't mean that you can't have instances where you will have to load components manually. In that case, you must provide all attributes that you will like to use manually. Attributes that you didn't provide will not be magically set from the manifest but you can make your life easier and map it like this.
```

Comment:
```suggestion
We created this cool way of importing attributes and loading components in your blocks or other components. However, that doesn't mean that you can't have instances where you have to load components manually. In that case, you have to provide all the attributes that you want to use manually. The attributes that you didn't provide will not be magically set from the manifest, but you can make your life easier and map it like this.
```

Sorry, I'm replacing _must_ with _have to_ because it sounds softer but it's giving an understanding that it is necessary. It's not a must(pun intended) but I believe it's a bit nicer to read. 🙂